src/requestor/app.py

```diff
@@ -25,7 +25,7 @@
             "{}/.gen3/requestor".format(os.path.expanduser("~")),
         ]
         config.load(search_folders=CONFIG_SEARCH_FOLDERS)
-except:
+except Exception:
```

Comment:
Is it currently catching anything else other than just `Exception` or is it a precaution?

Reply:
it's good practice to use `except Exception:` instead of just `except:` to avoid catching exceptions that are not meant to be caught

> A bare except: clause will catch SystemExit and KeyboardInterrupt exceptions, making it harder to interrupt a program with Control-C, and can disguise other problems. If you want to catch all exceptions that signal program errors, use except Exception: (bare except is equivalent to except BaseException:).

https://peps.python.org/pep-0008/#programming-recommendations